<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtMainWidget</class>
 <widget class="QWidget" name="QtMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtMainWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 14pt &quot;宋体&quot;;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnFirstWidgetByManal">
       <property name="text">
        <string>手动添加QWidget第一个Demo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFirstLayoutByManal">
       <property name="text">
        <string>手动添加QVLayout第一个Demo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddWidgetByCode">
       <property name="text">
        <string>手动添加QWidget示例</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mpToolButton">
       <property name="text">
        <string>帮助</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>image/help.png</normaloff>image/help.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnLabel">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 255, 0);
color: rgb(0, 85, 255);
font: 75 14pt &quot;Arial Narrow&quot;;</string>
       </property>
       <property name="text">
        <string>QLabel示例</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnComboBox">
       <property name="text">
        <string>QComboBox 示例</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnQTabWidget">
       <property name="text">
        <string>QTabWidget</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnQTabManual">
       <property name="text">
        <string>手动添加QTabWidget</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnQStackWidget">
       <property name="text">
        <string>QStackedWidget</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnListWidget">
       <property name="text">
        <string>QListWidget</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnQListWidgetByModelView">
       <property name="text">
        <string>使用ModelView框架来加载ListWidget</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnQListWidgetByDelegate">
       <property name="text">
        <string>Delegate来加载QListWidget(ModelView框架进阶)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnModelView">
       <property name="text">
        <string>进入Model View模式</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MyApplication.qrc"/>
 </resources>
 <connections/>
</ui>
